Polar/README.md

```diff
@@ -1,3 +1,73 @@
 # Polar Datacube Generation
 
-Information dedicated to the generation of the Polar Datacube.
\ No newline at end of file
+Information dedicated to the generation of the Polar Datacube.
+
+## Cube generation process
+
+The cube generation process is divided in four phases:
+
```

Comment:
Please add a note about using cube-gen conda env to generate the cubes, and how to activate it.